subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -2565,10 +2598,57 @@ void radio_ar_configure(uint32_t nirk, void *irk, uint8_t flags)
 
 	NRF_AAR->ENABLE = (AAR_ENABLE_ENABLE_Enabled << AAR_ENABLE_ENABLE_Pos) &
 			  AAR_ENABLE_ENABLE_Msk;
+
+#if defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
+	aar_job.nirk = nirk;
+
+	/* Input, Resolvable Address Hash offset in the legacy or extended advertising PDU.
+	 * Radio packet pointer offset by 3 compared to legacy	NRF_AAR that took Radio packet
+	 * pointer value.
+	 */
+	aar_job.in[0].ptr = (uint8_t *)addrptr + 3U;
+	aar_job.in[0].length = 3U;
```

Comment:
Could we not defined a `#define LEGACY_AAR_OFFSET 3U` to use here? 

If all the 3's in this function refer to the same value, they should use a defined value instead of a literal value IMO